services/QuillLMS/client/app/bundles/PremiumHub/shared.tsx

```diff
@@ -44,3 +46,9 @@ export function mapItemsIfNotAll(selectedItems, allItems, mapKey = 'id') {
 
   return selectedItems.map(i => i[mapKey])
 }
+
+/* Should produce the same output as backend `app/services/payload_hasher.rb:run` */
+export function hashPayload(payload) {
+  const joinedPayload = payload.join('-')
```

Comment:
same here, you might want a more specific name for `payload` or type it, since this is typescipt.